cli/localci/core/ccache_stats.py

```
@@ -0,0 +1,40 @@
+
+    Uses CCACHE_DIR so stats reflect the cache used by jobs. Returns the
+    stdout of ``ccache -s``, or None if ccache is not installed or the
+    command fails (e.g. dir missing or not a ccache dir).
+    """
+    ccache = shutil.which("ccache")
+    if not ccache:
+        return None
+    if not host_ccache_dir.exists():
+        return None
+    try:
+        result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)